### PR TITLE
chore: retire BOPS v1 submission endpoints and events

### PR DESCRIPTION
**Pull request by jessicamcinchak**
We have the go ahead from BOPS to stop sending parallel application payloads and only submit via their v2 endpoint.

**Changes here:**
- Creates a single Hasura event when sending to BOPS rather than two
  - Also applies to submission events created by Invite to Pay payment requests
- `/send/bops` endpoint is removed, and `/send/bops-v2` is renamed to `/send/bops` for simplicity
- `/admin/session/:sessionId/bops` endpoint is removed (we use existing `/admin/session/:sessionId/digital-planning-application` endpoint to inspect v2 payload)
- Removes tests and mock data that were v1 specific

I'll open a followup PR to additionally clean up planx-core, but this one can merge anytime before that as we're no longer calling `$api.export.bopsPayload(sessionId)` & v1 submissions are already failing on file tag validation.

**Testing:**
- Go through a submission service & confirm a single BOPS event is created on Send
- Repeat using Invite to Pay (you'll have to successfully complete mock payment)
- Confirm that submission succeeds (valid digital planning payload and audit table entry)

### CODE REVIEW

**Comment by jessicamcinchak** [comment]
[[EPIC] Update BOPS submissions to use new Digital Planning Data payload](https://trello.com/c/L24kI8JK/2671-epic-update-bops-submissions-to-use-new-digital-planning-data-payload)

File: api.planx.uk/modules/send/bops/bops.ts [hunk 1]
```diff
@@ -30,134 +30,6 @@ const sendToBOPS = async (req: Request, res: Response, next: NextFunction) => {
...
-    return next(
-      new ServerError({
-        status: 400,
-        message: `Missing application payload data to send to BOPS`,
-      }),
-    );
-  }
-
   // confirm that this session has not already been successfully submitted before proceeding
   const submittedApp = await checkBOPSAuditTable(payload?.sessionId, "v2");
```
[the remark this answers is not in the document]
Comment (jessicamcinchak) [reply api.planx.uk/modules/send/bops/bops.ts:34]: Had this same thought, but I actually think this is a nice reminder that BOPS audit table will continue to have a mix of _two_ versions ?

**Review by DafyddLlyr — APPROVED** [review]
**Testing notes**
- [x] Go through a submission service & confirm a single BOPS event is created on Send
- [x] Repeat using Invite to Pay (you'll have to successfully complete mock payment)
- [x] Confirm that submission succeeds (valid digital planning payload and audit table entry)

All working as expected ✅

File: api.planx.uk/modules/send/bops/bops.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (DafyddLlyr) [reply api.planx.uk/modules/send/bops/bops.ts:34]: Yep this is fair! No issues with this remaining as-is 👍

**Pull request merged**